poco/tests/versions_test.py

```diff
@@ -0,0 +1,39 @@
+import poco.poco as poco
+from .abstract_test import AbstractTestSuite
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+
+class VersionsTestSuite(AbstractTestSuite):
+
+    def test_version(self):
+        with self.captured_output() as (out, err):
+            with self.assertRaises(SystemExit) as context:
+                self.run_poco_command("--version")
+            self.assertIsNotNone(context.exception)
+        self.assertIn(poco.__version__, out.getvalue().strip())
+
+    def test_version_update_to_beta_tester(self):
+        extra_config = dict()
+        extra_config['beta_tester'] = 'True'
+        self.init_poco_config(extra_config)
+
+        with self.captured_output() as (out, err):
+            with self.assertRaises(SystemExit) as context:
+                self.run_poco_command("--version")
+            self.assertIsNotNone(context.exception)
+        self.assertIn(poco.__version__, out.getvalue().strip())
+
+    @mock.patch('poco.poco.__version__', '0.98.10')
+    @mock.patch('poco.services.environment_utils.EnvironmentUtils.parse_version')
+    def test_version_compare(self, parse_version):
```

Comment:
érdemes lenne még egy 1.x-es meg 10.x-es tesztet is irni. most maradhat, mert nincs közel, de hosszútávon jó lenne ezekre felkészülni előre